### PR TITLE
Renames and cleanup

### PR DESCRIPTION
**Pull request by larkz**
- Renaming PR, + refactoring for deployment.

### CODE REVIEW

File: app/src/main/kotlin/Reversi/ReversiSolver.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package Reversi
+
+import ca.aqtech.mctreesearch4j.StatefulSolver
+import ca.aqtech.mctreesearch4j.StatelessSolver
```
Comment (JunTaoLuo) [inline app/src/main/kotlin/Reversi/ReversiSolver.kt:4]: We aren't using the Stateless one, we can remove this import.

**Review by JunTaoLuo — APPROVED** [review]

File: app/src/test/kotlin/GridWorldTests.kt [hunk 2]
```diff
@@ -79,14 +79,14 @@ class GridWorldTests {
             openWorldErrors.add(this.testGridWorld(this.worldFeaturesOpenSpace, this.correctSolutionsOpenSpace))
         }
 
-        assertTrue(openWorldErrors.average() < 0.15, "Error percentage for grid world solution, (open space) " + openWorldErrors.average().toString() + " must be less than: 0.05")
+        assertTrue(openWorldErrors.average() < 0.7, "Error percentage for grid world solution, (open space) " + openWorldErrors.average().toString() + " must be less than: 0.05")
```
Comment (JunTaoLuo) [inline app/src/test/kotlin/GridWorldTests.kt:82]: This change can be reverted after the test fix right?

**Review by JunTaoLuo — APPROVED** [review]
Looks good

File: app/src/test/kotlin/GridWorldTests.kt (hunk 2 above)
Comment (larkz) [reply app/src/test/kotlin/GridWorldTests.kt:82]: yes we must change this back

Comment (larkz) [reply app/src/test/kotlin/GridWorldTests.kt]: its been changed back to 15% and tests pass

**Comment by larkz** [comment]
Release 0.0.2

**Pull request merged**